.github/workflows/test.yml

```diff
@@ -118,6 +36,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install poetry
-      - name: Check poetry lock
+          poetry install
+      - name: pre-commit checks
         run: |
-          poetry check --no-interaction
+          poetry run pre-commit run -a
```

Comment:
huh, it looks like this yaml is not checked with pre-commit itself because it's missing final LF. I will have a look at the configuration

Reply:
a top-level pre-commit exclude `exclude: '.git|.tox|venv|.venv'` also matched `.github`. Fixed via [d50b0ee](https://github.com/florczakraf/boogie-stats/pull/191/commits/d50b0ee08021b91f94ad1b160b7cae726334e8bd)